### DISCUSSION TITLE
Can't Launch Editor.

### DISCUSSION THREAD

**Discussion by codesnip** [o3de/o3de#2370] [Help Wanted]
I compiled o3de.
When i open Editor.exe and press open MyProject Editor .
Editor loads with message "loading plugins" and then closes with no error messages.
My computer specs:
cpu ryzen 1600.  ddr4 16gb  gpu: radeon rx 480

Log file:
Logged at Thursday, July 22, 2021 23:59:09
FileVersion: 0.0.0.0
ProductVersion: 0.0.0.0
LogFile: @log@/Editor.log
ProjectDir: H:\o3de\MyProject
Executable: Editor.exe (debug: no
Exception Code: 0xC0000005
Exception Addr: 0x0033:0x00007FF8A96FD380
Exception Module: 
Exception Description: EXCEPTION_ACCESS_VIOLATION, Attempt to read from address 0x00000176

The memory could not be "read"

Call Stack Trace:
 7) 00007FF93761C247 (KERNELBASE) : UnhandledExceptionFilter
 6) 00007FF939975270 (ntdll) : memset
 5) 00007FF93995C776 (ntdll) : _C_specific_handler
 4) 00007FF93997217F (ntdll) : _chkstk
 3) 00007FF939921454 (ntdll) : RtlRaiseException
 2) 00007FF939970CAE (ntdll) : KiUserExceptionDispatcher
 1) C:\o3de\o3de\Gems\Atom\RHI\DX12\Code\Source\RHI\CommandQueue.cpp (259) : AZ::DX12::CommandQueue::GetPlatformQueue

How to fix this error?

**Comment by UsualCoder1** [comment]
Did you install last drivers ? I had similar issue with DX12 render pipeline on my old slow GTX 970 with old drivers.

**Comment by codesnip** [comment]
I switched to stabilization/2106 branch. Now everethyng works fine.
